Stanford/JumpstartEngineering/Install/Users/RichardUser.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * @file
+ * Abstract Task Class.
+ */
+
+namespace Stanford\JumpstartEngineering\Install\Users;
+/**
+ *
+ */
+class RichardUser extends \AbstractInstallTask {
+
+  /**
+   * Set the site name.
+   *
+   * @param array $args
+   *   Installation arguments.
+   */
+  public function execute(&$args = array()) {
+
+    $editor_role = user_role_load_by_name('site member');
```

Comment:
For the sake of less confusion can this variable be re-named to reflect the role that is being loaded?